### DISCUSSION TITLE
Error receiver flinkmetrics

### DISCUSSION THREAD

**Discussion by WagnerCarvalho** [open-telemetry/opentelemetry-collector-contrib#12112] [Q&A] [answered]
Guys I'm using oRelease **v0.54.0**, but it seems to me that receiver **[flinkmetrics](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/flinkmetricsreceiver)** is not found!

**Error**: failed to get config: cannot unmarshal the configuration: unknown receivers type "flinkmetrics" for "flinkmetrics" (valid values: [influxdb journald kafkametrics mongodbatlas opencensus postgresql awsxray mongodb prometheus zipkin zookeeper apache couchdb dotnet_diagnostics hostmetrics memcached wavefront otlp cloudfoundry googlecloudpubsub kubeletstats rabbitmq windowsperfcounters awsecscontainermetrics googlecloudspanner jmx receiver_creator splunk_hec syslog redis riak podman_stats sapm statsd carbon elasticsearch jaeger k8s_cluster k8s_events nginx collectd signalfx tcplog awscontainerinsightreceiver awsfirehose docker_stats skywalking udplog fluentforward kafka mysql expvar filelog prometheus_simple])

Has anyone had this error?

**Comment by WagnerCarvalho** [comment, marked as answer]
I think the solution would be to put the component in distribution, but I don't have access!

https://github.com/open-telemetry/opentelemetry-collector-releases/pull/159/files

**Reply by codeboten** [reply]
That's right, thanks for creating the PR for adding the component. Please sign the CLA and we can move the PR forward!

**Reply by WagnerCarvalho** [reply]
> Isso mesmo, obrigado por criar o PR para adicionar o componente. Por favor, assine o CLA e podemos avançar com o PR!
DONE => https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/IT-24273
